com.reprezen.swagedit.core/src/com/reprezen/swagedit/core/json/references/JsonReference.java

```diff
@@ -223,4 +216,19 @@ public static JsonPointer getPointer(ObjectNode node) {
         }
     }
 
+    /**
```

Comment:
Javadoc for `JsonReference.isMissing()` should be corrected:
```java
    /**
     * Returns true if the reference points to an existing JSON document and the pointer to an existing node inside that
     * document.
     * 
     * @param baseURI
     * @return true if the reference can be resolved.
     */
    public boolean isMissing(JsonDocument document, URI baseURI) {
        if (isInvalid()) {
            return false;
        }

        JsonNode resolved = resolve(document, baseURI);
        return resolved == null || resolved.isMissingNode();
    }
```
It returns true if the reference can**not** be resolved and the pointer points to an **inexistent** element.